README.md

```diff
@@ -75,6 +75,16 @@ To enable Google Anaytics, add the following lines to your Jekyll site:
 
 Google Analytics will only appear in production, i.e., `JEKYLL_ENV=production`
 
+--
+
+### Enabling Excerpts on the Home Page
+
+To display post-excerpts on the Home Page, simply add the following to your `_config.yml`:
+
+```yaml
+show_excerpts: enabled
```

Comment:
I'd prefer excerpts to be enabled by default. But made it an option because I didn't want Minima to suddenly change how existing sites would look.. We can make it enabled by default in the next major release.